_includes/m2/solution_validation.html

```diff
@@ -8,10 +8,46 @@ <h2 class="section-heading text-uppercase">{{ site.data.m2.solution_validation.t
       </div>
   
       <div class="row">
-        <div class="col col-lg-12 text-center">
-          <p>Customers are nice. We want many of them. Also don't write text here. Put text and picture paths in _data/m2.yml under the specific section :)</p>
+        <div class="col col-lg-12 text-justify">
+            <p>{{ site.data.m2.solution_validation.introtext }}</p>
+
+            {% for l in site.data.m2.solution_validation.interview %}
+              <div class="row">
```

Comment:
Let's not create rows inside of rows. Move the for each outside the parent row, so the rows generated in the for loop are at the same level with the row that contains the intro text.